src/shared/components/Button/myFirstButton.jsx

```diff
@@ -0,0 +1,47 @@
+//Made by Adam Copeland.
+
+import React, { Component } from 'react';
+import { Alert, View, Text, StyleSheet, TouchableOpacity } from 'react-native';
+
+
+export class Settings extends Component {
+  _onPressButton() {
+    Alert.alert('This is actually awesome!!')
+  }
+
+  render() {
+    const bColor  =   this.props.buttonColor    ||  '#639af2';
```

Comment:
this seems fine but you will see some errors when you using [Eslint](https://eslint.org/). Just to get used to the linter. It's fine now